DatePicker/DatePickerBase.js

```diff
@@ -183,10 +185,12 @@ const DatePickerBase = kind({
 
 	styles: {
 		css,
+		dateTimeCss,
```

Comment:
I don't think this works the way you using it. You should be able to safely drop this line.